src/utils/error.ts

```diff
@@ -721,6 +721,10 @@ export class AuthClientErrorCode {
     code: 'not-found',
     message: 'The requested resource was not found.',
   };
+  public static USER_DISABLED = {
+    code: 'user-disabled',
+    message: 'The user account has been disabled by an administrator.',
```

Comment:
Maybe we shouldn't specify it is disabled by an admin
'The user record is disabled.'